src/plugins/console/server/lib/spec_definitions/json/generated/indices.put_index_template.json

```diff
@@ -0,0 +1,17 @@
+{
+  "indices.put_index_template": {
```

Comment:
We probably need to add an accompanying request body override for this endpoint: https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-put-template.html#put-index-template-api-request-body

Reply:
Done